include/ConfReader.cc

```diff
@@ -0,0 +1,75 @@
+#include "ConfReader.h"
+#include <map>
+#include <vector>
+#include <string>
+#include <fstream>
+#include <iostream>
+#include "utils.h"
+
+std::string ConfReader::returnContext(std::string line)
+{
+    line.erase(0, 1);
+    line.erase(line.size() - 1, 1);
+    // std::cout<<line;
+    if (line == rd)
+        return rd;
+    if (line == pd)
+        return pd;
+    if (line == pe)
+        return pe;
+    if (line == m)
+        return m;
+    if (line == rld)
+        return rld;
+    return "";
+}
+
+void ConfReader::readFile(std::string fileName)
+{
+    std::ifstream file(fileName, std::ios::out);
+    if (!file.is_open())
+    {
+        std::cerr << "Error: Unable to open CSV file " << fileName << " for reading!" << std::endl;
```

Comment:
conf file is actually an `ini` file, not a csv file. The purpose of a conf reader is to construct a model